### ISSUE TITLE
Programmatically create tickets 

### ISSUE THREAD

**Issue by TurnrDev** [django-helpdesk/django-helpdesk#695]
I'm not sure if this has been asked before but; If I create a ticket via `Ticket.objects.create()`, will an email to sent out to the Assigned User?

**Comment by gwasser** [comment]
I don't think so; the code to send emails to users appears to be in `forms.py`, you have to submit the form, creating the object isn't enough. But it's also simple code if you want to take a look and copy that stuff into your own application.

There's an old ticket #264 to add signals support so it does happen a bit more automagically but unfortunately haven't gotten to that yet. Thanks!

**Comment by JayTurnr** [comment]
Thank you! I'll take a look at `forms.py` and see what I can do :+1:

**Comment by gwasser** [comment]
Great! Let me know how it goes. Maybe I'll leave this ticket open for now as a reminder to make an easier way to do this, or at least improve documentation.

**Comment by timthelion** [comment]
Another way of doing this is to simple send an email to your queue programatically ;)

**Comment by JayTurnr** [comment]
@timthelion but then the submitter email wouldn't be the end user?

**Comment by timthelion** [comment]
That's true. I guess it depends on what you're doing. If you're importing issues from github then simply emailing links to them works, if you want to do something with better integration you'll need to work with the form.

**Comment by TurnrDev** [comment]
I eventually got this working to some extent! Since the close source project I used the code for is going open source I'll get around to pushing some changes upstream if I remember

**Comment by gwasser** [comment]
Pushing changes here would be greatly appreciated!

**Comment by madhav-dhungana** [comment]
> I eventually got this working to some extent! Since the close source project I used the code for is going open source I'll get around to pushing some changes upstream if I remember

If you have gone through the changes if its possible to update, would be highly appreciated!

**Comment by TurnrDev** [comment]
Unfortunately, I can no longer find the code I wrote

**Comment by uhurusurfa** [comment]
See how it is done in the email notifications test:
[https://github.com/django-helpdesk/django-helpdesk/blob/main/helpdesk/tests/test_email_notifications.py](https://github.com/django-helpdesk/django-helpdesk/blob/main/helpdesk/tests/test_email_notifications.py)
